src/common/options.cc

```diff
@@ -6980,6 +6980,10 @@ static std::vector<Option> get_rbd_options() {
     .set_default(false)
     .set_description("whether to block writes to the cache before the aio_write call completes"),
 
+    Option("rbd_shared_cache_enabled", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
Nit: I wonder if the naming (throughout) should be along the lines of "parent_image_cache"

Reply:
Actually, i also am not sure this naming. If you want to use parent_image_cache naming, i will do this. Thanks.

Reply:
modify all naming from SharedReadOnlyCache to ParentCache.